lib/utils.js

```diff
@@ -463,6 +463,23 @@ function transformEachsCallbackArgs(callBack) {
   return callBack;
 }
 
+/**
+ * Determines if the argument to `get()` is positive or negative.
+ * @param {ASTNode} indexArg
+ * @returns {number} Defaults to 0
+ */
+function determineGetIndex(indexArg) {
+  const indexNode = indexArg[0];
+  let index = 0; // default value
+
+  if (indexNode && indexNode.value) {
+    index = indexNode.value;
+  } else if (indexNode && indexNode.operator && indexNode.argument && indexNode.argument.value) {
```

Comment:
This is effectively looking for a `UnaryExpression`, right? I think it would be better to use `j.UnaryExpression.check(indexNode)` instead.

Reply:
Awesome, didn't know about those functions!